### ISSUE TITLE
thanks for your job

### ISSUE THREAD

**Issue by vaeXu** [poier/PreView#4]
@poier Hello, when I first saw your code, I was deeply amazed. I would like to ask you about the test results after training. When did you provide the code for the test module, I look forward to your next job,thanks.

**Comment by poier** [comment]
Hi,
thanks. ;-)
But I'm not sure if I correctly understand your question.
The run script (`main_run.py`) evaluates the learned model on the test set (after training). (By default, it will also output some view predictions). You can also use the pre-trained model (`--no-train` flag; see README) to produce the results. (And of course you can find the collected results in the paper. See [project page](https://poier.github.io/PreView/).)
Hope this helps.
